src/main/webapp/resources/js/pages/admin/components/Admin.jsx

```diff
@@ -58,30 +64,46 @@ export default function Admin() {
         <AdminHeader />
         <Content>
           <Suspense fallback={<ContentLoading />}>
-            <Router>
-              <BasicStats path={`${DEFAULT_URL}/${ADMIN.STATISTICS}`} default />
-              <AdvancedStatistics
-                path={`${DEFAULT_URL}/${ADMIN.STATISTICS}/:statType`}
-              />
-              <AdminUsersPage path={`${DEFAULT_URL}/${ADMIN.USERS}`} />
-              <AdminUserGroupsPage
-                path={`${DEFAULT_URL}/${ADMIN.USERGROUPS}/*`}
-              />
-              <ClientListingPage path={`${DEFAULT_URL}/${ADMIN.CLIENTS}`} />
-              <AdminRemoteApiPage path={`${DEFAULT_URL}/${ADMIN.REMOTEAPI}`} />
-              <AdminRemoteApiDetailsPage
-                path={`${DEFAULT_URL}/${ADMIN.REMOTEAPI}/:remoteId`}
-              />
-              <AdminSequencingRunsPage
-                path={`${DEFAULT_URL}/${ADMIN.SEQUENCINGRUNS}`}
-              />
-              <AdminNcbiExportsPage
-                path={`${DEFAULT_URL}/${ADMIN.NCBIEXPORTS}`}
-              />
-              <AnnouncementAdminPage
-                path={`${DEFAULT_URL}/${ADMIN.ANNOUNCEMENTS}`}
-              />
-            </Router>
+            <Routes>
+              <Route path={DEFAULT_URL} element={<AdminContent />}>
+                <Route path={ADMIN.STATISTICS} element={<BasicStats />} />
+                <Route
+                  path={`${ADMIN.STATISTICS}/:statType`}
+                  element={<AdvancedStatistics />}
+                />
+                <Route path={ADMIN.USERS} element={<AdminUsersPage />} />
+
+                <Route
+                  path={`${ADMIN.USERGROUPS}/list`}
```

Comment:
When adding a new UserGroup, once the group is added it does not display the usergroup details. It just displays a blank div within the admin panel. This is because it's linking to the incorrect page (Users List).

The url looks like this after successful creation of a usergroup
`http://localhost:8080/admin/groups/list/undefined/3`

Reply:
Fixed in a3af4e5